spec/unit/provider/service/smf_spec.rb

```diff
@@ -69,6 +73,11 @@
       @provider.stubs(:svcs).raises(Puppet::ExecutionFailure.new("no svc found"))
       expect(@provider.status).to eq(:stopped)
     end
+    it "should return stopped for an incomplete service on Solaris 11" do
+      Facter.stubs(:value).with(:operatingsystemrelease).returns('11.3')
```

Comment:
Assuming this is 11.3 because that matches the customer scenario?

Reply:
Yup.